setup.py

```diff
@@ -509,9 +508,30 @@ def run(self):
                'pycbc.inject',
                'pycbc.frame',
                'pycbc.catalog',
+               'pycbc.ligolw',
+               'pycbc.ligolw.utils',
                ],
-     package_data = {'pycbc.workflow': find_package_data('pycbc/workflow'),
-	             'pycbc.results': find_package_data('pycbc/results'),
-                     'pycbc.tmpltbank': find_package_data('pycbc/tmpltbank')},
+    package_data = {'pycbc.workflow': find_package_data('pycbc/workflow'),
+                    'pycbc.results': find_package_data('pycbc/results'),
+                    'pycbc.tmpltbank': find_package_data('pycbc/tmpltbank')},
+    ext_modules = [
```

Comment:
Ughh.. can we not build these? I really don't like adding this in. Do they depend on lalsuite at all or are they fully self contained?

Reply:
This is self-contained C-code in glue (does not depend on lal). If we want to remove this, the best approach IMO would be to move to stage 2 of this and start figuring out why our codes are broken against the current ligolw. I don't want to do anything special here for something that is temporary. As this has been in pycbc-glue for some time, we've been doing this for a while.

Reply:
(Or self-contained C-code that's now in the ligolw module).